app-crypt/efitools/efitools-1.8.1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Tools for manipulating UEFI secure boot platforms"
+HOMEPAGE="https://git.kernel.org/cgit/linux/kernel/git/jejb/efitools.git"
+SRC_URI="https://git.kernel.org/cgit/linux/kernel/git/jejb/efitools.git/snapshot/v${PV}.tar.bz2 -> ${P}.tar.gz"
+
+LICENSE="GPL-2 LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="libressl"
+
+RDEPEND="!libressl? ( dev-libs/openssl:0= )
+	libressl? ( dev-libs/libressl:0= )
+	sys-apps/util-linux"
+
+DEPEND="${RDEPEND}
+	app-crypt/sbsigntool
+	dev-perl/File-Slurp-Unicode
+	sys-apps/help2man
+	sys-boot/gnu-efi
+	virtual/pkgconfig"
+
+S=${WORKDIR}/v${PV}
+
+src_prepare() {
+	# Respect users CFLAGS (-fno-stack-protector -ffreestanding must be set)
+	sed -i -e "s/CFLAGS.*=.*/CFLAGS=$CFLAGS -fno-stack-protector -ffreestanding \$(ARCH3264)/" Make.rules || die
```

Comment:
Wouldn't `+=` work? Inlining `$CFLAGS` is a bad idea since you're relying on CFLAGS not clashing with sed's `/` separator.

Reply:
+= seems not to work, as I am getting:
Make.rules:42: *** Recursive variable 'CFLAGS' references itself (eventually).  Stop.

I've made it another way. I've dropped CFLAGS= and append via flag-o-matic the needed CFLAGS. As the rest already does $(CFLAGS), this seems to work fine. hWould be this an alternative?

I've commited this now.